src/compiler/types.ts

```diff
@@ -6123,8 +6123,15 @@ namespace ts {
         type: ESMap<string, number | string>;  // an object literal mapping named values to actual values
     }
 
+    /* @internal */
+    export interface AlternateModeDiagnostics {
+        diagnostic: DiagnosticMessage;
+        options: ReadonlySet<string>;

```

Comment:
I meant this should be `getOptionsNameMap` and reuse those maps we create instead of creating new one

Reply:
Aha! I think I understand, thank you for this!

Reply:
(if 660a99e is not what you meant then I am thoroughly stumped and apologetic)